src/libsync/account.cpp

```diff
@@ -951,11 +951,11 @@ void Account::slotDirectEditingRecieved(const QJsonDocument &json)
 
             auto *directEditor = new DirectEditor(id, name);
 
-            foreach(auto mimeType, mimeTypes) {
+            for (const auto &mimeType : mimeTypes) {
                 directEditor->addMimetype(mimeType.toString().toLatin1());
             }
 
-            foreach(auto optionalMimeType, optionalMimeTypes) {
+            for (const auto &optionalMimeType : optionalMimeTypes) {
```

Comment:
I would constify the original container variables and not add std::as_const here